tests/test_pdf_chisquared.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright (c) 2024 zfit
+import numpy as np
+import pytest
+
+import zfit
+from zfit import Parameter
+from zfit.models.dist_tfp import ChiSquared
```

Comment:
Use `zfit.pdf` to access the pdf. And remove the imports, inside the test, you can directly use `zfit.pdf.name` or `zfit.Parameter`